lib/java/ydoc-server/src/main/java/org/enso/ydoc/Main.java

```diff
@@ -15,7 +18,19 @@ public static void main(String[] args) throws Exception {
 
     Sampling.init();
 
-    try (var ydoc = Ydoc.builder().build()) {
+    var ydocHost = System.getenv(YDOC_HOST);
+    var ydocPort = System.getenv(YDOC_PORT);
```

Comment:
Why do we define these environment variables, when they are not used anywhere?

Reply:
`Main` class exists only for development purposes. I.e. you can start the Ydoc server on a custom socket with
```sh
$ env YDOC_HOST=127.0.0.1 YDOC_PORT=30030 sbt ydoc-server/run
```

Reply:
Preferably prefixed with `ENSO_`?